rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -1928,6 +1928,28 @@ class LessCloudsActionExecutor : public IUnsyncedActionExecutor {
 	}
 };
 
+class RotateSkyActionExecutor : public IUnsyncedActionExecutor {
+public:
+	RotateSkyActionExecutor() : IUnsyncedActionExecutor("RotateSky",
+		"Rotates the sky around axis by angle") {
+	}
```

Comment:
Would be ideal to provide a trivial widget that listens to `/luaui RotateSky args` instead (as a general rule for all engine /commands), but this is relatively harmless so it's fine if it stays.

Reply:
I think I will unbind this action

Reply:
engine-defined synced/unsynced actions are problematic often in such that they can't respect good interoperability with lua defined actions (any engine action for that matter anyway).

I also agree avoiding engine-defined actions in favor of basecontent widgets that define useful actions is preferred.

Some actions might be too useful to not be defined by engine though